### ISSUE TITLE
scale control in way of taking full screen shot

### ISSUE THREAD

**Issue by rejon** [newpalmyra/newpalmyra-AR#16]
![b319b608-edd8-44ac-bc72-b33e78e56ee0](https://user-images.githubusercontent.com/59623/46262165-ffda5b80-c4c2-11e8-8ece-48b9cba56526.png)

**Comment by lurnid** [comment]
Yep, that is annoying. I was thinking that a good approach would be to have a navigation drawer, or a floating action button that hides any controls so it's just that one icon on the screen in the bottom corner in normal mode

**Comment by rejon** [comment]
Or super cool to have pure finger pinch and zoom, there is also not a simple path to knowing that you should click the screen once app loads.

Maybe impossible to not have some simple ui or overview drawer

Jon
